src/javascript/app/pages/trade/common.js

```diff
@@ -367,9 +367,7 @@ const commonTrading = (() => {
     };
 
     const reloadPage = () => {
-        Defaults.remove('market', 'underlying', 'formname',
-            'date_start', 'time_start', 'expiry_type', 'expiry_date', 'expirt_time', 'duration_units', 'diration_value',
-            'amount', 'amount_type', 'currency', 'prediction');
+        Defaults.remove('market', 'underlying', 'formname', 'date_start', 'time_start', 'expiry_type', 'expiry_date', 'expirt_time', 'duration_units', 'diration_value', 'amount', 'amount_type', 'currency', 'prediction');
```

Comment:
```suggestion
        Defaults.remove('market', 'underlying', 'formname', 'date_start', 'time_start', 'expiry_type', 'expiry_date', 'expiry_time', 'duration_units', 'diration_value', 'amount', 'amount_type', 'currency', 'prediction');
```
I know that this is something that you didn't change, but its obviously a type, is it possible for you to check it?

Reply:
I found another typo besides this one. Therefore, I've introduce a constant for this strings :)